mesh/nurbs.cpp

```diff
@@ -5377,4 +5552,144 @@ void NURBSPatchMap::SetBdrPatchDofMap(int p, const KnotVector *kv[],  int *okv)
    }
 }
 
+Table *NURBSExtension::GetGlobalElementDofTable()
```

Comment:
These functions (1, 2, and 3D) all appear to be copied exactly from `ParNURBSExtension`. Would it be simpler to move them to `NURBSExtension`, so that `ParNURBSExtension` inherits them from the base class, instead of having identical copies in the base and derived classes?